.github/workflows/ci.yaml

```diff
@@ -283,3 +283,17 @@ jobs:
       - name: "Build docs"
         if: ${{ env.MKDOCS_INSIDERS_SSH_KEY_EXISTS != 'true' }}
         run: mkdocs build --strict -f mkdocs.generated.yml
+
+  check-formatter-stability:
```

Comment:
It could make sense to split this out into its own workflow and only run it on changes to `ruff_formatter` and `ruff_python_formatter`

Reply:
yep absolutely, there's much improvement to still be done